### PR TITLE
[CMSP-903] WordPress 6.5 docs & release note

### PR DESCRIPTION
**Pull request by jazzsequence**
## Summary

**[Performant Translations on Pantheon](https://docs.pantheon.io/guides/wordpress-developer/performant-translations)** - A new guide for how to use and work with WordPress' Performant Translations (`.l10n.php` files) on Pantheon
**[Release Notes](https://docs.pantheon.io/release-notes)** - WordPress 6.5 release note

### Dependencies and Timing

**Release**:
- [ ] When ready
- [x] After date: March 26, 2024 (or when WordPress 6.5 releases)

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/guides/wordpress-developer/12-performant-translations.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+terminus wp -- <site>.<env> i18n make-php wp-content/languages
+terminus env:commit <site>.<env> --message="Generate .l10n.php files"
+```
+
+The example above does three things: 1) sets the environment to `SFTP` mode, 2) generates the `.l10n.php` files, and 3) commits the changes to the environment. You can then switch back to `Git` mode if you prefer.
+
+
+### Considerations
+
+* Running `i18n make-php` will generate `.l10n.php` files _based on the current_ `.mo` files. You might want to run this command after updating your language files to ensure the `.l10n.php` files are up-to-date first.
```
Comment (pwtyler) [inline source/content/guides/wordpress-developer/12-performant-translations.md:42]: Note here how to update language files too?

File: source/content/guides/wordpress-developer/12-performant-translations.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* The existing [known issur relating to WordPress Language Packs](/wordpress-known-issues#language-packs) still apply. As noted above, language packs can only be installed and updated in SFTP mode and then committed to the filesystem via the dashboard or Terminus.
```
Comment (pwtyler) [inline source/content/guides/wordpress-developer/12-performant-translations.md:43]:
```suggestion
* The existing [known issue relating to WordPress Language Packs](/wordpress-known-issues#language-packs) still apply. As noted above, language packs can only be installed and updated in SFTP mode and then committed to the filesystem via the dashboard or Terminus.
```

**Comment by pwtyler** [comment]
Would be good to link back to the MU release note that affects 6.5 fonts.

**Comment by jazzsequence** [comment]
Notes updated. Ready for final 👀 and ✅. Holding merge until 6.5 is released.

**Review by rwagner00 — APPROVED** [review]
LGTM

**Pull request merged**